02-hello-accessibility/projects/starter/app/src/main/res/layout/item_try_it_recipe.xml

```diff
@@ -0,0 +1,112 @@
+  ~ or information technology.  Permission for such use, copying, modification,
+  ~ merger, publication, distribution, sublicensing, creation of derivative works,
+  ~ or sale is expressly withheld.
+  ~
+  ~ This project and source code may use libraries or frameworks that are
+  ~ released under various Open-Source licenses. Use of those libraries and
+  ~ frameworks are governed by their own individual licenses.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  ~ THE SOFTWARE.
+  -->
+
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginVertical="8dp"
+    app:cardCornerRadius="@dimen/card_radius">
+
+  <FrameLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/design_default_color_error">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="end|center_vertical"
+        android:clickable="false"
+        android:drawablePadding="@dimen/drawable_padding"
+        android:drawableTint="@color/design_default_color_on_primary"
+        android:drawableTop="@drawable/ic_baseline_thumb_down_24"
+        android:focusable="false"
+        android:padding="@dimen/space_normal"
+        android:text="@string/shared_discard"
+        android:textColor="@color/design_default_color_on_primary" />
+  </FrameLayout>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/item_recipe_card"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="?colorPrimary">
+
+    <CheckBox
+        android:id="@+id/item_recipe_made"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/space_normal"
+        android:text="@string/recipe_detail_made_recipe"
+        android:textColor="?colorOnPrimary"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/item_recipe_title" />
+
+    <TextView
+        android:id="@+id/item_recipe_title"
+        android:layout_width="match_parent"
+        android:layout_height="100dp"
```

Comment:
This should probably be an `@dimen`

Reply:
I'm going to leave this one here as intentionally broken. TextViews shouldn't have a set height! When the font size is scaled up, it chops off content.